docs/Other SDL Features/Retrieving Vehicle Data/index.md

```diff
@@ -41,6 +41,8 @@ You will only have access to vehicle data that is allowed to your `appName` and
 | VIN | vin | The Vehicle Identification Number |||
 | Window Status | windowStatus | An array of window locations and approximate position | RPC v7.0+ ||
 | Wiper Status | wiperStatus | The status of the wipers: off, automatic off, off moving, manual interaction off, manual interaction on, manual low, manual high, manual flick, wash, automatic low, automatic high, courtesy wipe, automatic adjust, stalled, no data exists |||
+| Seat Occupancy | seatOccupancy | The status of the seats that show whether each seat is occupied and belted or not | RPC v7.1+ ||
```

Comment:
The list appears to be in alphabetical order please move Seat Occupancy and Climate Data to the correct spots